ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -140,10 +147,26 @@
       },
     },
     {
-      id: 'E6S Conflag Strike Knockback',
-      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4C10', capture: false }),
+      id: 'E6S Downburst Knockback 1',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4BFB', capture: false }),
       response: Responses.knockback(),
     },
+    {
+      id: 'E6S Downburst Knockback 2',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4BFC', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'E6S Conflag Strike',
```

Comment:
This might be just deleted. Give an opinion.

Reply:
or maybe players could get ready to gather (Mostly `T/H`, `DPS`)

Reply:
I think saying knockback ~5 seconds before the knockback is good.  Maybe a timeline trigger would be better?

Reply:
This does really NOTHING about knockback, if you are saying about `Conflag Strike`

Reply:
Hmmm.. this change log(?) might be confusious?.. 
I added new `E6S Downburst Knockback 1` and `E6S Downburst Knockback 2`.
I renamed `E6S Conflag Strike Knockback` to `E6S Conflag Strike` since this does nothing about knockback.
I changed `response: Responses.knockback()` to new `infoText` in `E6S Conflag Strike`.

Reply:
Ah, this was already mentioned on issue, #1107.